### DISCUSSION TITLE
Support for high fidelity image understanding as in GPT-V API

### DISCUSSION THREAD

**Discussion by adrianmargin** [microsoft/autogen#1920] [General]
Is is any plan to add support for detail level "high" in [gpt4v_formatter](https://github.com/microsoft/autogen/blob/78a5a6b20c0acc771baef1aa903c1d3761b11d7c/autogen/agentchat/contrib/img_utils.py#L159)? The results of MultimodalConversableAgent could be greatly improved just by setting "high" value for "detail".
See https://platform.openai.com/docs/guides/vision/low-or-high-fidelity-image-understanding

**Comment by jackgerrits** [comment]
cc @BeibinLi
